database/migrations/postgres/1733415120086267_deployments_autoincrement.up.sql

```diff
@@ -23,7 +23,23 @@ SELECT setval('deployments_version_seq', coalesce(max(version), 0) + 1, false) F
 ALTER TABLE IF EXISTS deployments
 ALTER COLUMN version SET DEFAULT nextval('deployments_version_seq');
 
-ALTER TABLE IF EXISTS deployments DROP CONSTRAINT IF EXISTS deployments_pkey;
+DO $$
```

Comment:
same, why are we changing an old migration? Please add a new migration instead

Reply:
Same.